src/blocks/Listing/Listing.scss

```
@@ -39,6 +39,10 @@
             color: $link;
         }
 
+        p {
+            color: black;
+        }
```

please use the variable color, 
```suggestion
        color: $black;
```

---

Thank you @zahraard changed to $black variable color